### PR TITLE
Unify the namespace pattern for density.go with the others

### PR DESCRIPTION
**Pull request by jayunit100**
Just a little housekeeping w/ namespaces for density , so that they (1) don't  reference or use default ns and (2) use the already existing ns creation util format `e2e-tests-%v-%v` (i.e. e2e-tests-densities-osijd982hje9f8he2f).

### CODE REVIEW

File: test/e2e/density.go [hunk 1]
```diff
@@ -54,7 +54,9 @@ var _ = Describe("Density", func() {
 		expectNoError(err)
 		minionCount = len(minions.Items)
 		Expect(minionCount).NotTo(BeZero())
-		ns = api.NamespaceDefault
+		ns_, err := createTestingNS("density", c)
```
Comment (wojtek-t) [inline test/e2e/density.go]: nit: Can you please change "ns_" to "testingNS"?

**Comment by wojtek-t** [comment]
Thanks - just one very minor nit.

**Comment by jayunit100** [comment]
Hi @wojtek-t @satnam6502 , okay, fixed the nit !

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**